ReText/__main__.py

```diff
@@ -92,6 +93,18 @@ def main():
 	fileNames = list(map(canonicalize, sys.argv[1:]))
 	previewMode = False
 	readStdIn = False
+    
+	if not QDBusConnection.sessionBus().registerService('org.ReText'):
+		print('Send file to main instance')
+		iface = QDBusInterface('org.ReText', '/', '', QDBusConnection.sessionBus())
+		msg = iface.call('openFileWrapper', sys.argv[1] if len(sys.argv) > 1 else "")
+		QDBusReply(msg)
+		sys.exit(1)
+
+	connection = QDBusConnection.sessionBus()
+	connection.registerObject('/', window, QDBusConnection.ExportAllSlots)
```

Comment:
This also exports some slots on `org.qtproject.Qt.QWidget` interface that should not be exposed publicly. Like `setWindowModified` and `setWindowTitle`. Is there any way to avoid that?